src/CookiePlugin.php

```diff
@@ -109,11 +110,11 @@ private function createCookie(RequestInterface $request, $setCookie)
 
             switch (strtolower($key)) {
                 case 'expires':
-                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value);
+                    $expires = \DateTime::createFromFormat(\DateTime::COOKIE, $value);
```

Comment:
looking at the test failure on hhvm, i wonder if hhvm can't handle dates that far in the future? 

which in turn leads me to think we should check the return value as [\DateTime::createFromFormat](http://php.net/manual/en/datetime.createfromformat.php) is one of those stupid functions that returns false instead of throwing an exception on failure. if we check for `false === $expires` we can throw an exception and put the $value in the message to say it was not converted to a date.